pandas/util/mixins.py

```diff
@@ -0,0 +1,28 @@
+"""
+Mixins which can't go to core.common or core.base due to circular imports
+"""
```

Comment:
the point here is that the entire ``StringMethods`` infrastructure is wrong. It should use ``PandasDelegate`` and such; this was never fixed. This will also fix all of the import issues.